### DISCUSSION TITLE
Handle multiple PV creation

### DISCUSSION THREAD

**Discussion by nabokihms** [topolvm/topolvm#867] [Q&A]
Hello, community. 

I'm creating 600 pods with PVs in parallel (this is the requirement of one of our clients). I'm trying to understand when the allocated size of the node will be updated.

As far as I can tell, there is a VG service that runs once every ten minutes and notifies others about updated status
https://github.com/topolvm/topolvm/blob/ecfadb5c768d026d6d18a5f2adb4344c2a58b4c9/internal/lvmd/embed.go#L35-L46

Can the scheduler accidentally mispredict the node because I already allocated all the space, but the notifier still hasn't finished the cycle and there are still outdated values for allocated capacity on a node?

**Comment by llamerada-jp** [comment]
It pains me to say this, but it is a known limitation. 
https://github.com/topolvm/topolvm/blob/main/docs/limitations.md#capacity-aware-scheduling-may-go-wrong
For now, the way to mitigate this is to create pods with enough intervals.

**Reply by nabokihms** [reply]
@llamerada-jp understood, thanks for the quick answer.
